src/wordpress/integration-group.php

```diff
@@ -0,0 +1,65 @@
+<?php
+/**
+ * WPSEO plugin file.
+ *
+ * @package Yoast\YoastSEO\WordPress;
+ */
+
+namespace Yoast\YoastSEO\WordPress;
+
+/**
+ * Manage integrations and registers hooks at the required moment.
+ */
+class Integration_Group implements Integration {
```

Comment:
It could be worth a consideration to split these Integration definitions of into a separate PR so we can start using these more broadly in the plugin right away. I do think that, going forward, this is a patter we'll want to use a lot more and doesn't necessarily have to be tied to Indexables.